web/site/pages/annual-report.vue

```diff
@@ -221,33 +220,29 @@ if (import.meta.client) {
             @error="handleFormErrorEvent"
           >
             <UFormGroup name="radioGroup">
+              <!-- label for visual -->
               <template #label>
-                <div class="flex items-start gap-1">
-                  <span>{{ $t('page.annualReport.form.agmStatus.question', { year: busStore.currentBusiness.nextARYear }) }}</span>
-                  <!-- TODO: investigate better i18n/mobile tooltip options -->
-                  <UTooltip
-                    ref="tooltipRef"
-                    :text="$t('page.annualReport.form.agmStatus.tooltip')"
-                    :popper="{ arrow: true, placement: 'auto' }"
-                    tabindex="0"
-                    @focus="() => tooltipRef?.onMouseEnter()"
-                    @blur="() => tooltipRef?.onMouseLeave()"
-                  >
-                    <UIcon
-                      name="i-mdi-info-outline"
-                      class="size-6 shrink-0 text-bcGovColor-activeBlue [@media(pointer:coarse)]:hidden"
-                    />
-                  </UTooltip>
-                </div>
+                <span>{{ $t('page.annualReport.form.agmStatus.question', { year: busStore.currentBusiness.nextARYear }) }}
+                  <SbcTooltip id="agm-status-tooltip" :text="$t('page.annualReport.form.agmStatus.tooltip')" />
+                </span>
               </template>
-
-              <URadioGroup
-                v-model="selectedRadio"
-                :legend="$t('page.annualReport.form.agmStatus.question', { year: busStore.currentBusiness.nextARYear })"
-                :options
-                :ui="{ fieldset: 'space-y-2', legend: 'sr-only' }"
-                :ui-radio="{ label: 'text-base font-medium text-bcGovColor-midGray dark:text-gray-200', wrapper: 'relative flex items-center' }"
-              />
+              <fieldset>
```

Comment:
use radio instead of radio group, this allows setting aria-describedby from the tooltip content in the label